### DISCUSSION TITLE
Breakpoints not working on production (NextJS/Vercel)

### DISCUSSION THREAD

**Discussion by YaGRRusso** [tailwindlabs/tailwindcss#11553] [Help] [answered]
**What version of Tailwind CSS are you using?**

Tailwindcss v3.3.2

**What build tool (or framework if it abstracts the build tool) are you using?**

NextJS v13.1.6

**What version of Node.js are you using?**

Node v18 LTS

**What browser are you using?**

Chrome

**What operating system are you using?**

Zorin (Ubuntu/Linux)

**Reproduction URL**

https://nexo-front-355qt1rau-nexo-team.vercel.app

**Describe your issue**

- The breakpoint style its applied only after i change the screen (like its applied with JS)

- When i change the screen, the breakpoint style is applied at the bottom of the browser element styles list
 
## Screen with 767px
- Only 3 styles
- Display must be: hidden

- Display is: hidden
![Captura de tela de 2023-07-05 17-52-34](https://github.com/tailwindlabs/tailwindcss/assets/61320909/948ffbe8-efd4-478b-a177-affd59a51b47)

## Screen with 769px (md breakpoint)
- 4 styles (now the breakpoint style is applied on the bottom os the styles list)
- Display must be: flex
- Display is: hidden (display hidden stay on top of the breakpoint style)
![Captura de tela de 2023-07-05 17-53-09](https://github.com/tailwindlabs/tailwindcss/assets/61320909/4fc8e5a8-9d81-4361-8f02-edc57f9cb664)

**Comment by mybearworld** [comment]
Could you share your tailwindconfig?

**Comment by tobyab** [comment]
Is this only refusing to work in production, or is it working ok locally?

**Comment by YaGRRusso** [comment, marked as answer]
Sorry, i forgot to tell but i found the solution to the problem, i'll leave it here in case someone goes through this in the future:

I was using my own company component package created using tailwind and, when the package style was imported to the project, he was interfering on the project style, i've discovered this looking to the build files.

I've solved this by importing the package style on my ***global.css*** with `@import 'xxx'` instead of `import url('xxx')`

_I don't know why, but import with url doesn't seems work well on Next._

Doing that i also had to change my google font import to `import { yyy } from 'next/font/google'` on ***_app.tsx*** instead of `import url('yyy')` on ***global.css***

**Reply by RajdeepDs** [reply]
Hey, @YaGRRusso I am also facing the same issue, the breakpoints are not working in the production environment deployed to Vercel. I am using Turborepo, Next.js. Can you please help me out. 
**This is showing:**
![image](https://github.com/tailwindlabs/tailwindcss/assets/70424923/601bdb18-d60e-46cb-80cc-deb002b723d2)

**This should be:**
![image](https://github.com/tailwindlabs/tailwindcss/assets/70424923/5ee9b05b-caf0-4376-9d7e-9e2dc1f1b2f8)

Here is the repo: https://github.com/RajdeepDs/Muse

Thank you,
